packages/cli/src/commands/apps/authorize.ts

```diff
@@ -19,15 +19,17 @@ export default class AppsAuthorizeCommand extends SmartThingsCommand<typeof Apps
 	]
 
 	static examples = [
-		'$ smartthings apps:authorize arn:aws:lambda:us-east-1:1234567890:function:your-test-app',
-		'',
-		'Note that this command is the same as running the following with the AWS CLI:',
-		'',
-		'$ aws lambda add-permission --region us-east-1 \\',
-		'    --function-name arn:aws:lambda:us-east-1:1234567890:function:your-test-app \\',
-		'    --statement smartthings --principal 906037444270 --action lambda:InvokeFunction',
-		'',
-		'It requires your machine to be configured to run the AWS CLI',
+		{
+			description: `authorize an app
+
+NOTE: This command is the same as running the following with the AWS CLI:
+
+$ aws lambda add-permission --region us-east-1 --function-name arn:aws:lambda:us-east-1:1234567890:function:your-test-app
+>  --statement smartthings --principal 906037444270 --action lambda:InvokeFunction
+
+It requires your machine to be configured to run the AWS CLI.`,
+			command: 'smartthings apps:authorize arn:aws:lambda:us-east-1:1234567890:function:your-test-app',
+		},
```

Comment:
I can put the command in the middle of a description if I use strings but there are a couple of problems with it:

* It strips away any double newlines so it's impossible to have any blank lines between anything, including the command itself and either part of the description.
* Even though the command gets colored gray in the help (via `smartthings apps:authorize --help`), it does not get indented. If we manually add indentation with a couple of spaces, the gray color goes away.

For example, the following code:

```
		'authorize an app\n' +
		'\n' +
		'  $ smartthings apps:authorize arn:aws:lambda:us-east-1:1234567890:function:your-test-app\n' +
		'\n' +
		'Note that this command is the same as running the following with the AWS CLI:\n' +
		'\n' +
		'$ aws lambda add-permission --region us-east-1 \\\n' +
		'    --function-name arn:aws:lambda:us-east-1:1234567890:function:your-test-app \\\n' +
		'    --statement smartthings --principal 906037444270 --action lambda:InvokeFunction\n' +
		'\n' +
		'It requires your machine to be configured to run the AWS CLI',
```

Results in this output.

```
  authorize an app
    $ smartthings apps:authorize arn:aws:lambda:us-east-1:1234567890:function:your-test-app
  Note that this command is the same as running the following with the AWS CLI:
  $ aws lambda add-permission --region us-east-1 \
      --function-name arn:aws:lambda:us-east-1:1234567890:function:your-test-app \
      --statement smartthings --principal 906037444270 --action lambda:InvokeFunction
  It requires your machine to be configured to run the AWS CLI
```

I have changed "It" to "This command" which seems more clear to me.

I also split the aws-equivalent command into three lines instead of just two because the first line was too long.